Web instructions: quotes are wrong in build chain for Windows

Minor thing but an annoyance. Instructions say things like:

# Windows in cmd.exe
set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
set GNUARMEMB_TOOLCHAIN_PATH="C:\gnu_arm_embedded"

It turns out the double quotes on the toolchain path don't work with CMake. It will say something like:

CMake Error at D:/Git/repo/vf/zephyr/cmake/toolchain/gnuarmemb.cmake:14 (message):
  Nothing found at GNUARMEMB_TOOLCHAIN_PATH:
  '"e:/toolchains/gnu_arm_embedded"'

Since there probably is something in that path, it takes a bit of inspection to realize the double quotes from the instructions have made it all the way into the CMake script and cause the error.

There are probably multiple instances of this issue, spread across different toolchain instructions.

--- a/doc/getting_started/toolchain_3rd_party_x_compilers.rst
+++ b/doc/getting_started/toolchain_3rd_party_x_compilers.rst
@@ -35,7 +35,7 @@ GNU ARM Embedded
 
       # Windows in cmd.exe
       set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-      set GNUARMEMB_TOOLCHAIN_PATH="C:\gnu_arm_embedded"
+      set GNUARMEMB_TOOLCHAIN_PATH=C:\gnu_arm_embedded
 
 Intel ISSM
 **********

